assets/src/edit-story/elements/text/index.js

```diff
@@ -36,15 +36,10 @@ export const defaultAttributes = {
   bold: false,
   fontFamily: 'Roboto',
   fontFallback: ['Helvetica Neue', 'Helvetica', 'sans-serif'],
-  fontWeight: 400,
   fontSize: 36,
-  fontStyle: 'normal',
   backgroundColor: createSolid(196, 196, 196),
-  color: createSolid(0, 0, 0),
```

Comment:
Don't forget to rework the color presets logic in case the color attributes get removed with this PR.

Reply:
Oh yes, thanks. Yeah, that work has to be done too! And you should be able to apply color presets for both entire text field or a selection, I assume. I'll make sure it works for preset panel too.

Reply:
Added to todo.

Reply:
Fixed in 03aa64b and 1b4f9de